php/libraries/NDB_Page.class.inc

```diff
@@ -743,20 +743,6 @@ class NDB_Page extends \LORIS\Http\Endpoint implements RequestHandlerInterface
             ->withBody(new \LORIS\Http\StringStream($this->display() ?? ""));
     }
 
-    /**
-     * This function can be overridden in a module's page to load the necessary
-     * resources to check the permissions of a user.
-     *
-     * @param User                   $user    The user to load the resources for
-     * @param ServerRequestInterface $request The PSR15 Request being handled
-     *
-     * @return void
-     */
-    public function loadResources(
```

Comment:
should this change be in here?

Reply:
`\Module::handle` calls `loadResources()` on all pages that are accessed through `https://loris/$modulename/$pagename`. NDB_Page extends Endpoint, so I moved the function to the base of the hierarchy. I could put an empty `loadResources()` in the `ProjectDescription` class instead, but I think it makes more sense to just move the definition up a level since it affects every Endpoint.

Reply:
okayy that makes sense but im worried about this breaking other things that we won't pick up cz most of the testing has been done

Reply:
i was thinking it could break if other classes extend Endpoint but don't have the loadResources function implemented but i see now that the function is not an abstract method so it shouldn't break anything

Reply:
why does an empty loadResources need to be defined in the first place?

Reply:
because it's called at https://github.com/aces/Loris/blob/main/php/libraries/Module.class.inc#L313